<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FRENCH-DEF</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame[objectName=&quot;home_frame&quot;] {
	background-color: rgb(255, 255, 255);
}

QListWidget[objectName=&quot;history_view&quot;] {
	background-color: rgb(255, 255, 255);
	padding: 10px;
}

QListWidget::item {
    color: #444444;
}

/* QScrollArea */

QScrollBar:vertical {
    background-color: #19232D;
    width: 12px;
    margin: 16px 2px 16px 2px;
    border: 1px solid #CCCCCC;
    border-radius: 4px;
}

QScrollBar::add-line:vertical {
    margin: 3px 0px 3px 0px;
    border-image: url(&quot;:/styles/arrow_down_disabled.png&quot;);
    height: 10px;
    width: 10px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover,
QScrollBar::add-line:vertical:on {
    border-image: url(&quot;:/arrow_down.png&quot;);
    height: 10px;
    width: 10px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    margin: 3px 0px 3px 0px;
    border-image: url(&quot;:/arrow_up_disabled.png&quot;);
    height: 10px;
    width: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover,
QScrollBar::sub-line:vertical:on {
    border-image: url(&quot;:/arrow_up.png&quot;);
    height: 10px;
    width: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical,
QScrollBar::down-arrow:vertical {
    background: #CCCCCC;
    border-radius: 4px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="screen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1010</width>
      <height>580</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QFrame" name="home_frame">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>165</y>
        <width>520</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>520</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid lightgrey;
border-radius: 8px;
/*background-color: rgb(255, 255, 255);*/</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="started">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>245</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 18px;
	padding: 10px;
	color: #ffffff;
	background-color: rgb(119, 127, 255);
	border: 2px solid #777FFF;
}

QPushButton:hover {
	background-color: #777DDD;
}

QPushButton:pressed {
	background-color: rgba(119, 122, 170, 254);
}
</string>
       </property>
       <property name="text">
        <string>GO TO SEARCH</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/assets/back-arrow.png</normaloff>:/assets/back-arrow.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>480</width>
         <height>55</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 14pt &quot;Umpush&quot;;
padding: 0 15px;</string>
       </property>
       <property name="text">
        <string>135 000 d√©finitions...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>95</y>
         <width>480</width>
         <height>55</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 14pt &quot;Umpush&quot;;
padding: 0 15px;</string>
       </property>
       <property name="text">
        <string>90 000 articles...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>480</width>
         <height>55</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 14pt &quot;Umpush&quot;;
padding: 0 15px;</string>
       </property>
       <property name="text">
        <string>92 000 synonymes...</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>144</y>
        <width>371</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../res.qrc">:/assets/book_lover.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>370</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>13</y>
         <width>210</width>
         <height>54</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(119, 127, 255);
padding: 6px 0;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
border: 2px solid #777FFF;</string>
       </property>
       <property name="text">
        <string>FRENCH</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>13</y>
         <width>120</width>
         <height>54</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(119, 127, 255);
padding: 10px 0;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;</string>
       </property>
       <property name="text">
        <string>DEF</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>530</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(114, 159, 207);
	border-radius: 10px;
}

QLabel:hover {
	background-color: rgba(64, 147, 191, 50);
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a style=&quot;text-decoration:none&quot; href=&quot;https://www.larousse.fr/&quot;&gt;&lt;span style=&quot; color:#777fff;&quot;&gt;LAROUSSE.FR&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1010</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid lightgrey;
border-top: none;
border-left: none;
border-right: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="toggle_theme">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>6</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-top-left-radius: 22px;
	border-bottom-left-radius: 22px;
	border: 1px solid lightgrey;
	padding: 10px;
	background-color: rgba(119, 127, 255, 10);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 100);
}

QPushButton:pressed {
	background-color: rgba(119, 127, 255, 30);
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/assets/bulb-1.png</normaloff>:/assets/bulb-1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="search_input">
       <property name="geometry">
        <rect>
         <x>109</x>
         <y>6</y>
         <width>591</width>
         <height>44</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid lightgrey;
	border-top-left-radius: 22px;
	border-bottom-left-radius: 22px;
	border-right: none;
	padding-left: 20px;
	padding-right: 20px;
}
</string>
       </property>
       <property name="placeholderText">
        <string>Entrez le mot √† d√©finir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_button">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>6</y>
         <width>195</width>
         <height>44</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans Light</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid lightgrey;
	border-top-right-radius: 22px;
	border-bottom-right-radius: 22px;
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 100);
}

QPushButton:pressed {
	background-color: rgba(119, 127, 255, 30);
}
</string>
       </property>
       <property name="text">
        <string>D√âFINITION</string>
       </property>
      </widget>
      <widget class="QPushButton" name="about">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>6</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-top-right-radius: 22px;
	border-bottom-right-radius: 22px;
	border: 1px solid lightgrey;
	padding: 10px;
	background-color: rgba(119, 127, 255, 10);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 100);
}

QPushButton:pressed {
	background-color: rgba(119, 127, 255, 30);
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/assets/info.svg</normaloff>:/assets/info.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="search_exit">
       <property name="geometry">
        <rect>
         <x>955</x>
         <y>6</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-top-right-radius: 22px;
	border-bottom-right-radius: 22px;
	border: 1px solid lightgrey;
	padding: 10px;
	background-color: rgba(119, 127, 255, 10);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 100);
}

QPushButton:pressed {
	background-color: rgba(119, 127, 255, 30);
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/assets/logout.png</normaloff>:/assets/logout.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="reset_search">
       <property name="geometry">
        <rect>
         <x>911</x>
         <y>6</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-top-left-radius: 22px;
	border-bottom-left-radius: 22px;
	border: 1px solid lightgrey;
	padding: 10px;
	background-color: rgba(119, 127, 255, 10);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 100);
}

QPushButton:pressed {
	background-color: rgba(119, 127, 255, 30);
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/assets/clear.png</normaloff>:/assets/clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="search_response">
      <property name="geometry">
       <rect>
        <x>269</x>
        <y>59</y>
        <width>741</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>59</y>
        <width>270</width>
        <height>521</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="history_view">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>270</width>
         <height>481</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>190</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans Light</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #AAAAAA;</string>
       </property>
       <property name="text">
        <string>HISTORIQUE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="filter_button">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>0</y>
         <width>40</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="toolTipDuration">
        <number>2</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid lightgrey;
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 100);
}

QPushButton:pressed {
	background-color: rgba(119, 127, 255, 30);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/assets/refresh.png</normaloff>:/assets/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="reset_history">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="toolTipDuration">
        <number>2</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid lightgrey;
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 100);
}

QPushButton:pressed {
	background-color: rgba(119, 127, 255, 30);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/assets/delete.svg</normaloff>:/assets/delete.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
